docker/run.sh

```diff
@@ -52,30 +52,44 @@ function run_tests() {
     setup $TEST
     delta=$((`date +%s` - $now))
 
-    send_metrics_to_datadog "setup" $delta
+    send_timing_metric_to_datadog "setup" $delta
 
     now=`date +%s`
     su cchq -c "../run_tests $TEST $(printf " %q" "$@")"
     delta=$((`date +%s` - $now))
 
-    send_metrics_to_datadog "tests" $delta
+    send_timing_metric_to_datadog "tests" $delta
+    send_counter_metric_to_datadog
 }
 
-function send_metrics_to_datadog() {
+function send_timing_metric_to_datadog() {
+    send_metric_to_datadog "travis.timings.$1" $2 "gauge"
```

Comment:
I think there's a way to tell datadog that a metric represents a time. At least some of the riak metrics show up as times (with time units that automatically scale based on the magnitude of the metric) when you put them in a graph. See metric `riak.node_put_fsm_time_95` for example. I'm not sure how to do this, but it would be cool if we could add units to these. Here's the datadog riak integration in case that helps: http://docs.datadoghq.com/integrations/riak/